scripts/buildDocs.js

```diff
@@ -100,6 +100,13 @@ components.forEach(component => {
   /* Snippet */
   if (component.snippet) {
     content += `### Usage\n`;
+    if (component.usesAnimated) {
```

Comment:
I prefer to avoid adding warnings at this point
I rather add specials notes after we understand all of the complexities of rendering components on web and then decide how to tackle it - fix them, add warnings, etc.. 

I personally ok that during the development some stuff won't work 

Maybe instead we can have a list somewhere that document issues with each component. WDYT?

Reply:
Maybe we can add a more general `snippetNote` \ `snippetCaution` \ `usageNote` (etc) that will have the component's specific error\bug?
This way we have the issue and a way to find all issues without an outside list.
WDYT?

Reply:
In theory if the component live code isn't working, Im not sure we want to show it.. 
For now there are only several components that don't work, right? let's document it somewhere, not sure I want to use the json for tracking stuff, someone might misuse it for actual notes

Reply:
As discussed, let's remove snippets from components that don't work 
and add a warning in the build script about these components